script/codebuild/build

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# Prints shell input lines as they are read.
+# set -v
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+###########
+# INSTALL #
+###########
+
+# Fail if we're not using php71+
+if ! (php --version | grep -q "PHP 7.") ; then
+    echo "You're not using PHP 7.1+, please install it."
+    php --version
+    exit 1
+fi
+
+# copy the certificate from aws parameter store so we can use cosmos-release inside CodeBuild
+# this is a temporal solution until we have an official way to provide CodeBuild with certs
+echo $BBC_CERTIFICATE | base64 --decode > /etc/pki/tls/certs/client.crt
+echo $BBC_CERTIFICATE_KEY | base64 --decode > /etc/pki/tls/private/client.key
+echo $COSMOS_REQUIRED_CA | base64 --decode >> /etc/pki/tls/certs/ca-bundle.crt
+
+if [[ -n "$OAUTH" ]]; then
```

Comment:
Do you know what is OAUTH used for? In blogs I know it is for gelicons but I was wondering what could require it here (just out of curiosity)

Reply:
there's a couple of BBC custom packages in the composer.json, bbc-rmp/cloudwatch-monitoringhandler for example